test/replay_driver.c

```diff
@@ -260,18 +259,12 @@ double rdb_check_adds_per_second(void)
 
     timer = clock();
     for (i = 0; i < REPLAY_NUM_TRIALS; i += 3) {
-        if (srtp_rdb_check(&rdb, i + 2) != srtp_err_status_ok)
-            ++failures;
-        if (srtp_rdb_add_index(&rdb, i + 2) != srtp_err_status_ok)
-            ++failures;
-        if (srtp_rdb_check(&rdb, i + 1) != srtp_err_status_ok)
-            ++failures;
-        if (srtp_rdb_add_index(&rdb, i + 1) != srtp_err_status_ok)
-            ++failures;
-        if (srtp_rdb_check(&rdb, i) != srtp_err_status_ok)
-            ++failures;
-        if (srtp_rdb_add_index(&rdb, i) != srtp_err_status_ok)
-            ++failures;
+        srtp_rdb_check(&rdb, i + 2);
+        srtp_rdb_add_index(&rdb, i + 2);
+        srtp_rdb_check(&rdb, i + 1);
+        srtp_rdb_add_index(&rdb, i + 1);
+        srtp_rdb_check(&rdb, i);
+        srtp_rdb_add_index(&rdb, i);
```

Comment:
This seems unrelated to the main topic of this PR, but still a good thing to fix.

Reply:
It only showed up with clang 15, so some what related